uniffi_bindgen/src/bindings/python/templates/RustBufferStream.py

```diff
@@ -1,6 +1,6 @@
 
 class RustBufferStream(object):
-    """Helper for structured reading of values from a RustBuffer."""
+    # Helper for structured reading of bytes from a RustBuffer
```

Comment:
Same question about the reasoning behind demoting this from a docstring to a comment